.github/workflows/workflow_manual_run.yml

```diff
@@ -0,0 +1,59 @@
+name: Workflow Manual Run 
+
+on:
+  workflow_dispatch:
+    inputs:
+      workflow-path:
+        description: 'Specify the workflow path to run'
+        required: true
+        default: 'transforms/universal/noop'
+      kfp_v2:
+        description: '1 if kfp version v2 should be used. Otherwise false.'
+        required: false
+        default: '0'
+      debug:
+         description: 'True if use debug mode.'
+         required: false
+         default: 'true'
+
+
+jobs:
+    test-kfp-workflow-run:
+        env:
+          KFPv2 : ${{ github.event.inputs.kfp_v1 }}
+          WORKFLOW_PATH: ${{ github.event.inputs.workflow-path }}
+          DEBUG_MODE: ${{ github.event.inputs.debug }}
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v4
+            - name: Free up space in github runner
+              # Free space as indicated here : https://github.com/actions/runner-images/issues/2840#issuecomment-790492173
+              run: |
+                  df -h
+                  sudo rm -rf "/usr/local/share/boost"
+                  sudo rm -rf "$AGENT_TOOLSDIRECTORY"
+                  sudo rm -rf /usr/share/dotnet /opt/ghc /usr/local/lib/android /usr/local/share/powershell /usr/share/swift /usr/lib/jvm /usr/local/.ghcup
+                  sudo docker rmi $(docker image ls -aq) >/dev/null 2>&1 || true
+                  df -h
+            - name: Test KFP worflow run
+              timeout-minutes: 120
+              run: |
+                  source kind/requirements.env
+                  export PATH=$PATH:/tmp/
+                  curl -Lo /tmp/kind https://kind.sigs.k8s.io/dl/v${KIND_VERSION}/kind-linux-amd64
+                  curl -fsSL -o /tmp/get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3
+                  chmod 700 /tmp/get_helm.sh
+                  HELM_INSTALL_DIR=/tmp/ /tmp/get_helm.sh -v v${HELM_VERSION} --no-sudo
+                  chmod 777 /tmp/helm
+                  chmod 777 /tmp/kind
```

Comment:
can you reorder command, so the chmod will be near the dir/file creation?

Reply:
Fixed. Thanks